sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryIO.java

```diff
@@ -1162,8 +1161,9 @@ private void executeQuery(
 
     private JobConfigurationQuery createBasicQueryConfig() {
       return new JobConfigurationQuery()
-          .setQuery(query.get())
           .setFlattenResults(flattenResults)
+          .setPriority("BATCH")
```

Comment:
Reverted this change. I think this will lead to inconsistent behavior across runners which violates user expectation, but I'm tired.